examples/smart-mastering-complete/src/main/ml-modules/root/custom-modules/custom/dob-match.xqy

```diff
@@ -1,33 +1,31 @@
-xquery version "1.0-ml";
-
-module namespace algorithms = "http://marklogic.com/smart-mastering/algorithms";
-
-import module namespace helper-impl = "http://marklogic.com/smart-mastering/helper-impl"
-at "/com.marklogic.smart-mastering/matcher-impl/helper-impl.xqy";
-
-declare namespace matcher = "http://marklogic.com/smart-mastering/matcher";
-
-declare option xdmp:mapping "false";
-
-(:
-Example of a simple DOB matching algorithm that expands the query by bumping each date up by one year.
-:)
-declare function algorithms:dob-match(
-  $expand-values as xs:string*,
-  $expand-xml as node(),
-  $options-xml as node()
-) as cts:query*
-{
-  let $property-name := $expand-xml/entityPropertyPath
-
-  (: A custom match action is no longer being passed the weight, so it's hardcoded here. DHFPROD-6324 captures the issue. :)
-  let $weight := 10
-
-  for $value in $expand-values
-  where $value castable as xs:date
-  return
-    let $query := helper-impl:property-name-to-query($options-xml, $property-name)($value, $weight)
-    let $next-year := xs:date($value) + xs:yearMonthDuration("P1Y")
-    let $next-query := helper-impl:property-name-to-query($options-xml, $property-name)(xs:string($next-year), $weight)
-    return cts:or-query(($query, $next-query))
-};
+xquery version "1.0-ml";
+
+module namespace algorithms = "http://marklogic.com/smart-mastering/algorithms";
+
+import module namespace helper-impl = "http://marklogic.com/smart-mastering/helper-impl"
+at "/com.marklogic.smart-mastering/matcher-impl/helper-impl.xqy";
+
+declare namespace matcher = "http://marklogic.com/smart-mastering/matcher";
+
+declare option xdmp:mapping "false";
+
+(:
+Example of a simple DOB matching algorithm that expands the query by bumping each date up by one year.
+:)
+declare function algorithms:dob-match(

```

Comment:
Nice - this means that this function will still work without being modified?

Reply:
Yes, I reverted to the version in the 5.2 branch.